deploy/efs-csi/01-daemonset.yaml

```diff
@@ -0,0 +1,99 @@
+# Source: https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/51d19a433dcfc47fbb7b7a0e1c8ff6ab98ce87e9/deploy/kubernetes/base/node.yaml
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: efs-csi-node
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: efs-csi-node
+  template:
+    metadata:
+      labels:
+        app: efs-csi-node
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ''
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      tolerations:
+        - operator: Exists
+      containers:
+        - name: efs-plugin
+          securityContext:
+            privileged: true
+          image: amazon/aws-efs-csi-driver:latest
```

Comment:
This should be a fully qualified image name

Reply:
What repo should we use? Is docker.io acceptable?

Reply:
`registry.hub.docker.com/amazon/aws-efs-csi-driver:latest` - We may want to pin the version here too @2uasimojo so they we can be sure each install is the same, use the image we used for testing.

Reply:
We should also note that if we are using docker up, we will have AWS China issues with this. cc: @dofinn @c-e-brumm

Reply:
@cblecker what is the alternative? Quay proxy?

Reply:
There's a docker hub mirror in china, but it would require an operator on the cluster to detect it's in china and change the image

Reply:
We have not conducted a speed test of dockerhub.io from china but we have pulled images from there successfully. To be certain though, is there a way we can toggle this on/off for regions like china?

Reply:
Thus far, docker appears to be more reliable then quay.

Reply:
If we push to a repo we managed on quay.io we can control the version available to OSD and have a single registry to work with for OSD in China.